CONTRIBUTING.md

```diff
@@ -90,5 +112,5 @@ Make sure to credit all relevant people in `pack.mcmeta`. Most modules credit th
 
```

Comment:
If `pack.mcmeta` doesn't exist anymore, how do credits with for each pack? Maybe some parts of the CONTRIBUTING.md are outdated now?

Reply:
You're right I still need to update these sections

Reply:
I'd leave a description here about what exactly goes in `meta.gm4`. It be the same as before but you can describe what you need and what you don't need and perhaps make it clear on what each field *expects*.

Reply:
Made these changes in https://github.com/Gamemode4Dev/GM4_Datapacks/pull/828/commits/d5150c1dec57675cea8fc5952af91dbb7e81166a, let me know if things are unclear or missing.